### ISSUE TITLE
Broken on arch linux without user intervention

### ISSUE THREAD

**Issue by fiskhest** [kevinhwang91/fzf-tmux-script#6]
After following the installation instructions and pressing prefix+w, I was presented with an empty window:

![Screenshot_2020-09-20-33_3200x1750](https://user-images.githubusercontent.com/26885267/93720095-2483de00-fb87-11ea-833c-11ea96c67617.png)

During troubleshooting, I struck two different issues

```
    panes_info=$(tmux list-panes -aF \
        '#D #{s| |_|:#{=|9|…:session_name}} #I.#P#{?window_zoomed_flag,⬢,❄} #{pane_tty} #{pane_current_path} #T' |
        sed -E "/^$cur_id /d")
    ttys=$(awk '{printf("%s,", $4)}' <<<$panes_info | sed 's/,$//')
```
`ttys` somehow translated to what was in `#{pane_current_path}`, not `#{pane_tty}`. This is because the second column of `panes_info` `#{s| |_|:#{=|9|…:session_name}}` actually isn't being populated. When I try `tmux display-message -p '#{s| |_|:#{=|9|…:session_name}}'`  I get an empty result. I couldn't figure out how to format that to get the same result you are getting, so I worked around it by replacing with `#{session_name}` (I currently don't use spaces when naming sessions)

Fixing that, next issue:

```
    ps_info=$(ps -t$ttys -o stat,pid,pcpu,mem,thcount,time,tname,cmd |
        awk '$1~/\+/ {$1="";print $0}')
```

Here, I received an issue `error: unknown user-defined format specifier "mem"`. Consulting the docs of the regular `ps` bundled on arch linux, I replaced `mem` with `pmem` and it finally seems like my installation somewhat replicates the use case screencasts.

![Screenshot_2020-09-20-16_3200x1750](https://user-images.githubusercontent.com/26885267/93720288-5a759200-fb88-11ea-91fc-8e616c79d49d.png)

**Comment by kevinhwang91** [comment]
Thanks for your feedback.

The first issue:
I'm using the master branch of tmux, this issue is fixed by https://github.com/tmux/tmux/commit/d0957529edcdd3f616d0361972dd819b8b4c29d0 which is next-3.3 version, please forgives my laziness for not testing 
 https://github.com/kevinhwang91/fzf-tmux-script/commit/a6d0793d0d825b957a6d63e1f2ecd20706e85b90 in 3.0 version.

The second issue:
I guess you mix your code into the script during debuging. The source code is `pmem` not `mem`.
https://github.com/kevinhwang91/fzf-tmux-script/blob/e3708a81b9c60842efcba95a41fe483884cd3a15/panes/fzf-panes.tmux#L144

**Comment by kevinhwang91** [comment]
I have pushed the patch, please update the latest code.

**Comment by fiskhest** [comment]
> Thanks for your feedback.
> 
> The first issue:
> I'm using the master branch of tmux, this issue is fixed by [tmux/tmux@d095752](https://github.com/tmux/tmux/commit/d0957529edcdd3f616d0361972dd819b8b4c29d0) which is next-3.3 version, please forgives my laziness for not testing
> [a6d0793](https://github.com/kevinhwang91/fzf-tmux-script/commit/a6d0793d0d825b957a6d63e1f2ecd20706e85b90) in 3.0 version.

No worries, I am using 3.1b and was under the impression it's fairly bleeding edge, but TIL :)

> 
> The second issue:
> I guess you mix your code into the script during debuging. The source code is `pmem` not `mem`.
> 
> https://github.com/kevinhwang91/fzf-tmux-script/blob/e3708a81b9c60842efcba95a41fe483884cd3a15/panes/fzf-panes.tmux#L144

oh yeah, seems like I wasn't paying attention when working around the first issue, you are correct.
